docs/operator-manual/Rancher-Fleet.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_label: "Rancher Fleet"
+title: ""
+---
+
+# Managing Kubewarden with Rancher Fleet
+
+The Kubewarden Helm charts, as other Helm charts, can be managed via [Rancher
+Fleet](https://fleet.rancher.io/). Rancher Fleet uses Kubernetes CRDs to define
+a GitOps approach for managing Kubernetes clusters. It does this by defining
+Fleet Bundles.
+
+The Kubewarden charts are normal charts, and they have defined dependencies that
+need to be ready before the dependent chart is deployed:
```

Comment:
Do we want to interlink to existing documentation for each of these points?

Reply:
I personally think that would be a bit too much, I would however put a link to the quickstart installation docs

Reply:
that sounds better, done in https://github.com/kubewarden/docs/pull/161/commits/eae9a1cf5bf6643303c549ca2199971d47ae197c.